### PR TITLE
Active storage

### PR DESCRIPTION
**Pull request by johnckealy**
Added fields for input of animal and active storage capability (multiple images are selectable). Connected to cloudinary with my cloudinary account. Will send auth key over slack.

### CODE REVIEW

File: app/views/animals/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+
+<h1>Add the details of your animal</h1>
+
+<div class="container">
+  <%= simple_form_for(@animal,  html: { class: "form-group w-50 p-5 mx-auto"}) do |f| %>
+    <%= f.input :name %>
+    <%= f.input :species %>
```
Comment (kathrin298) [inline app/views/animals/new.html.erb]: Set the label to something else than species as we discussed before?

File: app/views/animals/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <%= f.input :hourly_rate %>
+    <%= f.input :category, collection: Animal::CATEGORIES %>
+    <%= f.input :gender, collection: Animal::GENDERS %>
+    <%= f.input :photos, as: :file, input_html: { multiple: true } %>
+    <%= f.submit %>
+  <% end %>
```
Comment (kathrin298) [inline app/views/animals/new.html.erb]: missing other non-mandatory input fields: description, place of origin, age

Comment (johnckealy) [reply app/views/animals/new.html.erb]: done

File: app/views/animals/new.html.erb (hunk 1 above)
Comment (johnckealy) [reply app/views/animals/new.html.erb]: done

**Pull request merged**